internal/build/pom.xml

```diff
@@ -868,18 +863,59 @@
     </build>
 
     <profiles>
-        <!-- Set a module name for the parent that is explicitly not inherited -->
         <profile>
-            <id>in-parent</id>
+            <id>jdbi.jpms-auto-module</id>
             <activation>
+                <property>
+                    <name>packaging</name>
+                    <value>jar</value>
+                </property>
                 <file>
-                    <exists>.jdbi-parent</exists>
+                    <missing>src/main/java/module-info.java</missing>
```

Comment:
Why do we want conditional auto-module activation? I would have thought it would be an auto-module in main branch, and then as part of the module PR work we change it to not use auto-modules at all.

Reply:
because right now, we don't have any explicit modules, so we ship everything with auto modules. This is refactoring the existing code (removing the .jdbi-parent touch files).

The explicit JPMS code is still a long way out. This allows us to slowly fade it in (jdbi maven modules with a module-info will use that, all the others will use automatic module names)